src/drivers/power_monitor/ina226/ina226.cpp

```diff
@@ -754,58 +585,6 @@ stop()
 	return PX4_OK;
 }
 
```

Comment:
We still need the test to vet the chip. Listener produces to much data

Reply:
Listener updated. https://github.com/PX4/Firmware/pull/12113